pkg/fuzzer/stats.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package fuzzer
+
+const (
+	statGenerate       = "exec gen"
```

Comment:
Let's keep stats as integers as they are now. Strings are slower, require indirections and require to use maps instead of arrays, which makes everything even slower, generates more garbage and requries to use mutexes for any stats operations.
While now with integers we have a small, nice, dense array with elements that can be updated and read atomically.

Reply:
As discussed offline, we'll keep using a map for now.